modules/db/src/main/java/org/mule/module/db/internal/config/domain/query/ParameterizedQueryTemplateFactoryBean.java

```diff
@@ -143,4 +158,10 @@ public boolean isSingleton()
     {
         return false;
     }
+
+    private String buildNotDefinedInParamErrorMessage(String name)
+    {
+        return format("Parameter with name %s, used in the query text, does not match any defined query parameter name.", name);
```

Comment:
Parameter with name %s, used in the query text, does not match any defined query parameter name >
Parameter with name '%s', used in the query text, does not match any defined query parameter name defined in the query template